doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -298,14 +305,22 @@ Bluetooth samples
   * :ref:`bluetooth_central_hids`
   * :ref:`peripheral_hids_keyboard`
 
-* Updated the configurations of the non-secure ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples to properly use the TF-M profile instead of the predefined minimal TF-M profile:
+* Updated
+    * The configurations of the non-secure ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples to properly use the TF-M profile instead of the predefined minimal TF-M profile:
 
-  * :ref:`bluetooth_central_hids`
-  * :ref:`peripheral_hids_keyboard`
-  * :ref:`peripheral_hids_mouse`
+      * :ref:`bluetooth_central_hids`
+      * :ref:`peripheral_hids_keyboard`
+      * :ref:`peripheral_hids_mouse`
+
+      This change results from the Bluetooth subsystem transition to the PSA cryptographic standard.
+      The Bluetooth stack can now use the PSA crypto API in the non-secure domain as all necessary TF-M partitions are configured properly.
+    * The configurations of the following samples by increasing the main thread stack size (:kconfig:option:`CONFIG_MAIN_STACK_SIZE`) to prevent stack overflows:
```

Comment:
```suggestion
      The Bluetooth stack can now use the PSA crypto API in the non-secure domain as all necessary TF-M partitions are configured properly.

    * The configurations of the following samples by increasing the main thread stack size (:kconfig:option:`CONFIG_MAIN_STACK_SIZE`) to prevent stack overflows:
```